### PR TITLE
Comicinfo metadata

### PR DESCRIPTION
**Pull request by Gin-no-kami**
Added metadata plugin to parse info out of the ComicInfo.xml file embedded inside of archives.

### CODE REVIEW

**Comment by Gin-no-kami** [comment]
I want to be upfront about this merge request. I did my testing of this plugin through manual verification after uploading the Plugin to my LANraragi instance (run with Docker). I included verification tests similar to the other metadata plugins with the expected values, but I don't know to run them.

**Comment by Difegue** [comment]
np, we can just run the tests here! 👍 Thanks a lot for putting the work in on this.

File: lib/LANraragi/Plugin/Metadata/ComicInfo.pm [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+            $stringxml .= $line;
+        }
+
+        #Parse file into DOM object and extract tags
+        my $genre;
+        my $group;
+        my $url;
+        my $artist;
+        my $lang;
+        my $result = Mojo::DOM->new->xml(1)->parse($stringxml)->at('Genre');
```
Comment (Difegue) [inline lib/LANraragi/Plugin/Metadata/ComicInfo.pm:60]: You could technically only do the XML parsing once and store the DOM object to reuse later:  

```
my $dom = Mojo::DOM->new->xml(1)->parse($stringxml);

if (defined $dom->at('Genre')) {
  $genre = $dom->at('Genre')->text;
}
# etc
```  

Not a big deal though, fine to merge without that fixed.

**Review by Difegue — APPROVED** [review]
Looks perfect! I have one minor nitpick but I think I'll merge this regardless as it's not a big deal.  

Thanks again, it's nice to finally have ComicInfo support after all this time 🙏  

@holopin-bot @Gin-no-kami

**Comment by Difegue** [comment]
welp the bot doesn't work in reviews I guess
@holopin-bot @Gin-no-kami

**Pull request merged**